azure-batch/azure/batch/batch_service_client.py

```diff
@@ -23,6 +23,7 @@
 from .operations.task_operations import TaskOperations
 from .operations.compute_node_operations import ComputeNodeOperations
 from . import models
+from .custom.patch import patch_client
```

Comment:
Do we need to re-add this every single time we regenerate?

I guess there's no other good way...?
Is there a way we can easily know we forgot to do this? I guess the tests will fail...

Reply:
Yea the tests will fail.  They said they are working on adding it to the auto-generation though. So short term tests will fail. Long term it will be done automatically

Reply:
@matthchr It's in my roadmap to support a "patch.py" file and "custom", it's why I suggested it in the first place.